src/components/SelectorGroup/SelectorGroup.story.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2019, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+
+import SelectorGroup from './SelectorGroup';
+
+export default {
+  title: 'Forms/Selector/SelectorGroup',
+  component: SelectorGroup,
+  parameters: {
+    jest: ['SelectorGroup']
+  }
+};
+
+const options = [
+  {
+    children: 'Apple',
+    value: 'apple'
+  },
+  {
+    children: 'Banana',
+    value: 'banana'
+  },
+  {
+    children: 'Mango',
+    value: 'mango'
+  }
+];
+
+/* eslint-disable react/prop-types */
+const SelectorGroupWithState = ({ children, ...props }) => {
+  const [value, setValue] = useState(props.multiple ? [] : '');
+  const handleChange = event => {
+    event.persist();
+    setValue(prev => {
+      if (!props.multiple) {
+        return event.target.value;
+      }
+      return prev.includes(event.target.value)
+        ? prev.filter(v => v !== event.target.value)
+        : [...prev, event.target.value];
+    });
+  };
```

Comment:
What about always using an array?
So that we don't have to pass `multiple`, if this component grows the number of specific props can grow with it.... 🙂 
If we always pass an array it can have just 1 element or multiple.
Inversion of control pattern 🙂

Reply:
The `multiple` prop determines if multiple options can be selected. We need this information to decide whether we should render radio buttons or checkboxes under the hood.

When only a single option can be selected, I would find it confusing and cumbersome if the value was an array. Having said that, it's actually up to you how you want to store the state. The component simply calls the `onChange` handler with the clicked option. What you do with that information is up to you 😊